utils/Cargo.toml

```diff
@@ -28,7 +28,7 @@ rand = "0.8.0"
 rand_core = "0.6.3"
 
 [dev-dependencies]
-ark-ec = { version = "0.3.0", features = [ "parallel" ] }
+ark-ec = { version = "0.4.0", features = [ "parallel" ] }
```

Comment:
I'm a bit surprised to see it here, had an impression upgrading to `0.4.0` is quite nontrivial.

Also, do utilities have their separate set of versions?..

Reply:
I seem to have tinkered with this crate first, realized upgrading ark was out of scope, and forgot to ever return to it when I thought workspace deps were a good idea- nice catch, fixing

Reply:
@emberian makes sense (including in other comments), happy to merge after it's fixed!